Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2049 @@
+										},
+										"url": {
+											"raw": "{{baseUrl}}{{Monitoring}}/component/:uuid",
+											"host": [
+												"{{baseUrl}}{{Monitoring}}"
+											],
+											"path": [
+												"component",
+												":uuid"
+											],
+											"variable": [
+												{
+													"key": "uuid",
+													"value": "{{UUIDComponentResponse}}",
+													"description": "(Required) Uuid of the component properties being updated"
+												}
+											]
+										},
+										"description": "This operation updates a component in FRAIHMWORK. The uuid of the component to be operated upon is supposed to be established as a result of running the initial POST of the component."
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "DELETE",
+							"item": [
+								{
+									"name": "DELETE Component",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													"\r",
+													"\r",
+													""
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"auth": {
+											"type": "bearer",
+											"bearer": [
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "DELETE",
+										"header": [],
+										"url": {
+											"raw": "{{baseUrl}}{{Monitoring}}/component/:uuid",
+											"host": [
+												"{{baseUrl}}{{Monitoring}}"
+											],
+											"path": [
+												"component",
+												":uuid"
+											],
+											"variable": [
+												{
+													"key": "uuid",
+													"value": "{{UUIDComponentResponse}}",
+													"description": "(Required) Uuid of the component properties being deleted"
+												}
+											]
+										},
+										"description": "This operation removes a component from FRAIHMWORK. The uuid of the component to be operated upon is supposed to be established as a result of running the initial POST of the component."
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "/refresh",
+							"item": [
+								{
+									"name": "Refresh component Copy",
+									"event": [
```

Comment:
remove "Copy" from request name

Reply:
Addressed! File scrubbed for all instances of word 'copy'